template/audit-history.pug

```diff
@@ -0,0 +1,2 @@
+| <{_element}div{/} id="{$_id}" class="{_ui}ui{/} {$class} {_class}{/}" style="{$style}" {$attributes}>{$Content}
```

Comment:
why first char is pipe? See: https://github.com/atk4/ui/blob/develop/template/semantic-ui/element.pug

Reply:
fixed